sql/hive/src/test/scala/org/apache/spark/sql/hive/parquetSuites.scala

```diff
@@ -716,6 +717,39 @@ class ParquetSourceSuite extends ParquetPartitioningTest {
     }
   }
 
+  test("Alias used in converted data source tables") {
```

Comment:
Yeah, I can remove it. 

You are right. That example is just for showing the outcome of the code changes. Thanks!